libs/platform/src/lib/components/menu-button/menu-button.component.scss

```diff
@@ -0,0 +1,7 @@
+.fdp-menu-btn-truncate__txt {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    display: inline-block;
+    vertical-align: bottom;
+}
```

Comment:
I might be wrong but the truncation behavior is coming from Fundamental-styles. If it's not it's probably better to add it there instead of having it here

Reply:
will make a separate PR for this. will raise PR in this week.

Reply:
Thank you!

Reply:
I'm seeing the following misalignment of the menu in relation to the button:

<img width="339" alt="Screen Shot 2020-03-25 at 3 53 28 PM" src="https://user-images.githubusercontent.com/48103491/77593119-e09e6d80-6eb0-11ea-8083-17e163239c3b.png">

You need to add the following CSS or else the menu will not be positioned properly at the bottom of the menu button:
```
:host {
    display: inline-block;
}
```

Reply:
Thanks, had missed it. included above css.